src/features/hotspots/setup/HotspotSelectionScreen.tsx

```diff
@@ -28,6 +28,9 @@ const HotspotSetupSelectionScreen = () => {
           <HotspotSelectionCard hotspotType="NEBRAIN" />
           <HotspotSelectionCard hotspotType="NEBRAOUT" />
         </Box>
+        <Box flexDirection="row" height={191}>
```

Comment:
Can you add a `width="50%"` here until another maker comes in so that its the same width as the other buttons.

It will look like this:
![Screen Shot 2021-02-19 at 9 58 05 PM](https://user-images.githubusercontent.com/6345962/108585460-93638f80-72fd-11eb-80c5-8a15bddd99b8.png)